lib/fog/storage/google_json/models/file.rb

```diff
@@ -26,6 +26,24 @@ class File < Fog::Model
 
         # attribute :body
 
+        # https://cloud.google.com/storage/docs/access-control#predefined-acl
```

Comment:
This link doesn't work. I believe https://cloud.google.com/storage/docs/access-control/lists#predefined-acls is what you wanted.